python/mxnet/amp/lists/symbol_bf16.py

```diff
@@ -21,8 +21,13 @@
 # Functions that should be cast to lower precision
 BF16_FUNCS = [
     'Convolution',
```

Comment:
This whole file still needs to be updated - though I am not sure whether I should do it in this PR, since this may take some time, to verify which ops properly support `bf16`.